pkg/koordlet/util/cgroup_parent_utils.go

```diff
@@ -49,6 +50,12 @@ func GetKubeQoSByCgroupParent(cgroupDir string) corev1.PodQOSClass {
 	return corev1.PodQOSGuaranteed
 }
 
+// @return like kubepods-burstable.slice/kubepods-pod7712555c_ce62_454a_9e18_9ff0217b8941.slice/
+// /sys/fs/cgroup/blkio/kubepods.slice/kubepods-burstable.slice/kubepods-pod7712555c_ce62_454a_9e18_9ff0217b8941.slice
+func GetPodCgroupBlkIOAbsolutePath(podParentDir string) string {
+	return path.Join(system.Conf.CgroupRootDir, system.CgroupBlkioDir, podParentDir)
```

Comment:
A minor issue: Please use `filepath. Join` and check the other modifications where the `path` pkg is still adopted.

Reply:
There are many other people's codes that also use the path.Join function, do I need to change them all at once?

Reply:
I am curious why we use filepath package?

Reply:
Not required in this patch. Feel free to do the cleanup in another PR.